### PR TITLE
Added the 'focus_on_fit' variable to exclude entities from fitting map and more

### PR DESCRIPTION
**Pull request by baeksj**
1. focus _on_fit - when adding entities such as zones to the map, there may be cases where you want to exclude these entities from the fit map target.
2. Add 'Reset focus' button on the map. This button icon uses the @mdi package
3. Set padding value to 0.1 when fitting the map.
4. Lexical fix for gradualOpacity in README.md
5. Add z_index_offset for Overlapping markers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/nathan-gs/ha-map-card#readme",
   "dependencies": {
+    "@mdi/js": "^7.4.47",
```
Comment (nathan-gs) [inline package.json:22]: You can use mdi icons natively in Home Assistant, using `ha-icon`.

Comment (baeksj) [reply package.json]: Removed the dependency on the @mdi/js package.

Comment (baeksj) [reply package.json]: Sorry for the multiple commits.

**Comment by baeksj** [comment]
I'll submit a new pull request later with the bug fix

**Pull request closed without merging**